.travis.yml

```diff
@@ -31,13 +31,16 @@ before_install:
   - sudo add-apt-repository ppa:ubuntugis/ubuntugis-unstable -y
   - sudo apt-get update -q
   - sudo apt-get install binutils libproj-dev gdal-bin -y
+  - pip install -U pip setuptools wheel
+  - pip install -r requirements-test.txt
```

Comment:
not sure why the isort install is failing in the build, try adding -U:
```
pip install -U -r requirements-test.txt
```

Reply:
I used the -U flag in command but still install in failing in the build. Perhaps due to old versions of python that requires lower version of isort